### PR TITLE
add details on kuzzle login/logout and jwtToken

### PR DESCRIPTION
**Pull request by stafyniaksacha**

### CODE REVIEW

File: source/includes/_kuzzleObject.md [hunk 1]
```diff
@@ -965,6 +967,7 @@ Returns the `Kuzzle` object to allow chaining.
 #### Callback response
 
 Resolves to the `Kuzzle` object itself once the login process is complete, either successfully or not.  
+The `Kuzzle` object will have the property `jwtToken` filled if login success.
```
Comment (scottinet) [inline source/includes/_kuzzleObject.md]: Instead of `if login success`: `if logging in succeeds`

File: source/includes/_kuzzleObject.md [hunk 2]
```diff
@@ -1004,6 +1007,8 @@ Logout the user.
 |---------------|---------|----------------------------------------|
 | ``callback`` | function | Optional callback handling the response |
 
+The logout method will revoke availability of `jwtToken`
```
Comment (scottinet) [inline source/includes/_kuzzleObject.md]: `This method empties the 'jwtToken' property`

File: source/includes/_kuzzleObject.md [hunk 3]
```diff
@@ -1011,6 +1016,51 @@ Returns the `Kuzzle` object to allow chaining.
 #### Callback response
 
 Resolves to the `Kuzzle` object itself once the logout process is complete, either successfully or not.  
+The `Kuzzle` object will unset the property `jwtToken` if logout success.
```
Comment (scottinet) [inline source/includes/_kuzzleObject.md]: `if logging out succeeds`

File: source/includes/_kuzzleObject.md [hunk 4]
```diff
@@ -1011,6 +1016,51 @@ Returns the `Kuzzle` object to allow chaining.
...
+
+
+## getJwtToken
+
+```js
+var jwtToken = kuzzle.getJwtToken();
+```
+
+```java
+String jwtToken = kuzzle.setJwtToken("eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiYWRtaW4iOnRydWV9.TJVA95OrM7E2cBab30RMHrHDcEfxjoYZgeFONFh7HgQ");
```
Comment (scottinet) [inline source/includes/_kuzzleObject.md]: Copy-pasted example from the `setJwtToken` method documentation

File: source/includes/_kuzzleObject.md [hunk 5]
```diff
@@ -1011,6 +1016,51 @@ Returns the `Kuzzle` object to allow chaining.
...
+```
+
+Get internal jwtToken used to request kuzzle.
+
+#### setJwtToken(jwtToken)
```
Comment (scottinet) [inline source/includes/_kuzzleObject.md]: `getJwtToken`

File: source/includes/_kuzzleObject.md (hunk 3 above)
Comment (xbill82) [inline source/includes/_kuzzleObject.md]: `if the user is successfully logged out`

**Comment by xbill82** [comment]
:+1:

File: source/includes/_kuzzleObject.md (hunk 3 above)
Comment (scottinet) [inline source/includes/_kuzzleObject.md]: Better :-)

**Comment by scottinet** [comment]
+1

**Pull request merged**